### DISCUSSION TITLE
File Peeker in Typebot

### DISCUSSION THREAD

**Discussion by SolluSaaS** [minio/minio#20271] [Q&A] [answered]
Hello, everyone.
It would be very interesting if, in the Typebot integration with Minio, there was the possibility of selecting a file that was previously sent to the Minio bucket. In some cases, we end up with a lot of duplicate files in the bucket, simply because there's no way to select files that were sent at other times or in other flows, like logos, for example.
It's an idea.

![image](https://github.com/user-attachments/assets/0cf89076-f201-454c-9b79-94f14f09adf3)

I'm not sure if this is the best place to talk about this or if I should be discussing it on Typebot's Github, and I don't have much experience with Github interactions.
If I'm making a mistake, please forgive me.
Brotherly hugs...

**Comment by marktheunissen** [comment, marked as answer]
Hi @SolluSaaS this issue is definitely best discussed with the Typebot team, MinIO is a storage layer and can easily allow Typebot to "select a file that was previously sent to the Minio bucket", this is just listing a bucket and then reading the file. Please open a ticket with Typebot.
